python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/class-maps.j2

```diff
@@ -23,6 +23,12 @@
 {%         elif class_map.ip.access_group is arista.avd.defined %}
 {%             set row.type = 'acl' %}
 {%             set row.value = class_map.ip.access_group %}
+{%         elif class_map.dscp is arista.avd.defined %}
```

Comment:
All this would be simpler implemented by just checking for the var and print out the line directly. You can then do elif on all the first ones and then nest dscp and ecn under an `else` block and check those with a regular `if` inside.

Reply:
Done!